Makefile

```diff
@@ -27,6 +27,14 @@ app-static-analysis: ## to run static analysis
 
 app-test: ## to run unit tests
 	vendor/bin/phpunit
+	@for filename in $$(find snippets -type f -name *.php); do \
```

Comment:
should be in a dedicated target, because not called during CI here (Travis calls `make ci`)